test/Grid.spec.js

```diff
@@ -1,12 +1,13 @@
 import Adapter from 'enzyme-adapter-react-16';
-import { expect } from 'chai';
+import { assert, expect } from 'chai';
 import Grid from '../src/Grid/Grid';
 import Paper from 'material-ui/Paper';
 import PropTypes from 'prop-types';
 import React from 'react';
-import sinon from 'sinon';
+import sinon, { spy } from 'sinon';
 import Enzyme, { mount, shallow } from 'enzyme';
 import Table, { TableBody, TableCell, TableHead, TableRow } from 'material-ui/Table';
+import Axios from 'axios';
```

Comment:
Why do you need this? If you need use Axios use https://www.npmjs.com/package/axios-mock-adapter

Reply:
I'm going to remove the unused import